README.md

```diff
@@ -1,8 +1,49 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# John Cao . Jarvis Consulting
 
+I am a recent graduate from York University's Lassonde School of Engineering with a Bachelor's degree in Computer Science. I also hold a Bachelor's degree in Chemical Engineering from the University of Toronto. During my time at York, I have worked as part of the school's IT department where I acted as the main source of support for several business applications. I also spent 16 months working as an intern at Hydro One Telecom where I was able to speed up numerous tasks and processes through building scripts in Python and Bash. Through these various experiences, I was able to demonstrate my problem solving and communication skills. My interest in pursuing a career in software started when I took an Introduction to Programming course during my first undergrad. I was drawn to how programming allowed me to create and build things in a very accessible manner. Just a couple of lines of code and I was able to see the results of my work. Currently I am an Applications and Data Engineer at Jarvis where I have had the opportunity to improve my level of proficiency in various tools and technologies such as Java, Linux, and Docker as well as learning about good software development practices. I am looking forward to bring my skills and knowledge and contribute to your team.
+
+
+## Skills
+
+**Proficient:** Java, Python, Linux/Bash, RDBMS/SQL (PostgreSQL), Agile/Scrum, Git
+
+**Competent:** HTML/CSS, Docker, Maven, REST API, Django
+
+**Familiar:** JavaScript, JUnit, Scikit-learn, Machine Learning, Computer networking
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_JohnCao](https://github.com/jarviscanada/jarvis_data_eng_JohnCao)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JohnCao/tree/master/linux_sql)]: Implemented a Linux cluster monitoring solution using bash shell scripts that collect hardware specifications of each Linux server and their resource usage. The data collected is stored in a PostgreSQL database provisioned using Docker which can then be queried to answer business questions.
```

Comment:
# Please include the following keywords: CentOS, Bash/Shell, docker, RDBMS/SQL/Postgresql, crontab, data analytics